xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxLendingServiceRaw.java

```diff
@@ -0,0 +1,277 @@
+package org.knowm.xchange.ftx.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.knowm.xchange.ftx.FtxAdapters;
+import org.knowm.xchange.ftx.FtxExchange;
+import org.knowm.xchange.ftx.dto.account.*;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FtxLendingServiceRaw extends FtxBaseService {
+
+  public FtxLendingServiceRaw(FtxExchange exchange) {
+    super(exchange);
+  }
+
+  public FtxLendDataDto stopLending(String subaccount, String coin) {
+    return lend(subaccount, coin, 0, 0);
+  }
+
+  public List<FtxLendDataDto> stopLending(String subaccount, List<String> coins) {
+    return coins.stream()
+        .map(coin -> stopLending(subaccount, coin))
+        .collect(Collectors.toList());
+  }
+
+  public List<FtxLendDataDto> stopLending(String subaccount, String... coins) {
+    return stopLending(subaccount, Arrays.asList(coins));
+  }
+
+  public FtxLendDataDto lendAll(String subaccount, String coin, double rate) {
+    Objects.requireNonNull(coin);
```

Comment:
remove this line, again was not addressed

Reply:
Method was removed